Adding options to prombench's Makefile.

saketjajoo opened the pull request:
The options include commands to create and delete the core infrastructure as well as to apply cluster-wide resources.

saketjajoo committed:
Adding options to prombench's Makefile.

The options include commands to create and delete the core infrastructure as well as to apply cluster-wide resources.

Signed-off-by: saketjajoo <saketjajoo77@gmail.com>

bwplotka reviewed (approved):
Thanks! I wonder if it makes sense to update docs as well accordingly 🤔 (https://github.com/prometheus/test-infra/blob/77e3913332ee861df991e85bf5ad8118b2f933fa/prombench/README.md#setup-and-run-prombench)

saketjajoo committed:
Updating the prombench docs to reuse the commands in Makefile.

Instead of having separate commands to create the core infrastructure
for various cloud providers, the commands from the Makefile can be
resued. This commit changes the docs accordingly to reflect this.

Signed-off-by: saketjajoo <saketjajoo77@gmail.com>

bwplotka reviewed (approved):
Amazing!

Thanks 💪🏽
